src/tests/libecalc/input/validation/snapshots/test_validation_json_schemas/test_json_schema_changed/schemas.json

```diff
@@ -1472,6 +1563,51 @@
                     ],
                     "title": "Temperature",
                     "type": "object"
+                },
+                "YamlTrain_YamlCompressor_": {
+                    "additionalProperties": false,
+                    "properties": {
+                        "CATEGORY": {
+                            "description": "User defined category",
+                            "title": "CATEGORY",
+                            "type": "string"
+                        },
+                        "NAME": {
+                            "description": "Consumer name",
+                            "title": "NAME",
+                            "type": "string"
+                        },
+                        "STAGES": {
+                            "items": {
+                                "$ref": "#/definitions/YamlCompressor"
+                            },
+                            "title": "Stages",
+                            "type": "array"
+                        },
+                        "STREAMS": {
+                            "description": "List of streams",
+                            "items": {
+                                "$ref": "#/definitions/YamlStream"
+                            },
+                            "title": "Streams",
+                            "type": "array"
+                        },
+                        "TYPE": {
+                            "default": "TRAIN_V2",
+                            "description": "The type of the component",
+                            "enum": [
+                                "TRAIN_V2"
+                            ],
+                            "title": "Type",
+                            "type": "string"
+                        }
+                    },
+                    "required": [
+                        "NAME",
+                        "STAGES"
+                    ],
```

Comment:
Should STREAMS be required as well? Not much happening without "pipes" connecting the stages somehow...?

Reply:
I initially implemented default streams, so not required, but I removed it/postponed it. So we'll see, I guess